tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2920,6 +2925,13 @@ def rgb_to_yiq(images):
 
   Returns:
     images: tensor with the same shape as `images`.
+
+    Usage Example:
+    ```python
+   
+    >>> x = tf.random.normal(shape=(256, 256, 3))
```

Comment:
Likewise, no random image but instead something where the numbers are easy to understand